### PR TITLE
feat(pulumi): Add support for suppress-outputs, debug, and json comma…

### PR DESCRIPTION
**Pull request by ffMathy**
…nd-line arguments.

### CODE REVIEW

File: packages/pulumi/src/executors/up/up.impl.ts [hunk 1]
```diff
@@ -24,7 +27,10 @@ export default async function creatExecutor(
       'pulumi up', 
       options.stack && `--stack=${options.stack}`,
       options.skipPreview && '--skip-preview',
-      options.yes && '--yes'
+      options.yes && '--yes',
+      options.suppressOutputs && '--suppress-outputs',
+      options.debug && '--debug',
```
Comment (TriPSs) [inline packages/pulumi/src/executors/up/up.impl.ts:32]: Quesing that `--debug` logs more? If so can we change this to the constant `USE_VERBOSE_LOGGING` from `@nx-extend/core`

Comment (ffMathy) [reply packages/pulumi/src/executors/up/up.impl.ts:32]: That's a great idea! Unfortunately, I am pressed on time these days. Feel free to slip that in there too if you don't want to wait ❤️

Comment (TriPSs) [reply packages/pulumi/src/executors/up/up.impl.ts:32]: Thanks, added these change to #139.

**Comment by TriPSs** [comment]
Changes added #139

**Pull request closed without merging**